### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MariamTurky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,357 @@
...
+
+## Requirements
+
+Install the needed packages by the following command. You might need to adapt the cuda versions for
+torch and torchvision specified in *requirements.txt*.
+Find a pytorch installation guide for your
+system [here](https://pytorch.org/get-started/locally/#start-locally).
+
+````shell
+pip install -r requirements.txt
```
Comment (helrewaidy) [inline README.md:119]: where is the requirements.txt?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,357 @@
...
+````
+
+Among others, this repository is mainly built on the following packages.
+You may want to familiarize yourself with their basic use beforehand.
+
+- **[Pytorch](https://pytorch.org/)**: The machine learning/deep learning library used in this
+  repository.
+- **[Black](https://black.readthedocs.io/en/stable/)**: Code Formatter used in this Repo. Check out the [official docs](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html) on how to use Black or [here](https://akshay-jain.medium.com/pycharm-black-with-formatting-on-auto-save-4797972cf5de) on how to integrate Black into PyCharm.
```
Comment (helrewaidy) [inline README.md:127]: Really! Black is the second most important package in this repo not MONAI, or optuna?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,357 @@
...
+<a href="https://app.clear.ml"><img alt="Signup" src="https://img.shields.io/badge/Clear%7CML-Signup-brightgreen"></a>
+<a href="https://black.readthedocs.io/en/stable"><img alt="L: Hydra" src="https://img.shields.io/badge/Code Style-Black-black" ></a>
+</div>
+
+This repository contains a fully Automated Benchmarking System For Medical Imaging Segmentation, which facilitates the comparison of state-of-the-art models' performance.
+Our system enables architecture selection from a diverse range of models and automates the optimization of loss functions and parameter tuning through different search space algorithms. It also integrates a centralized logging system that tracks and shares all experiment-related details during training in real-time dashboards.
+we have explored various model architectures encompassing different families, including the [nnU-Net](https://arxiv.org/abs/1809.10486), which automatically adapts to the dataset fingerprint, the [AttentionU-Net](https://arxiv.org/abs/1804.03999) utilizes an attention mechanism for feature extraction, and [UNETR](https://arxiv.org/abs/2103.10504).
```
Comment (helrewaidy) [inline README.md:13]: nnunet is not here yet, right?

**Review by helrewaidy — COMMENTED** [review]
Nice work! 
Although its not covering everything, but its a good start

File: README.md (hunk 3 above)
Comment (MariamTurky) [reply README.md:13]: yes

File: README.md (hunk 2 above)
Comment (MariamTurky) [reply README.md:127]: i have handled

File: README.md (hunk 1 above)
Comment (MariamTurky) [reply README.md:117]: i have uploaded this

File: README.md (hunk 3 above)
Comment (MariamTurky) [reply README.md]: yes

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,255 @@
+absl-py==1.4.0
```
Comment (helrewaidy) [inline requirements.txt:1]: include direct dependencies only. if you're exporting from conda env, you might want to use `--from-history` param

File: README.md [hunk 5]
```diff
@@ -1,2 +1,355 @@
...
+````
+
+#### [Start using ClearML](https://clear.ml/docs/latest/docs/getting_started/ds/ds_first_steps) 
+
+The ClearML run-time components:
+
+* The ClearML Python Package - for integrating ClearML into scripts by adding just two lines of code, and optionally extending experiments and other workflows with ClearML's powerful and versatile set of classes and methods.
+* The ClearML Server - for storing experiment, model, and workflow data; supporting the Web UI experiment manager and MLOps automation for reproducibility and tuning. It is available as a hosted service and open source to deploy ClearML Server.
+
+<img src="https://raw.githubusercontent.com/allegroai/clearml-docs/main/docs/img/clearml_architecture.png" width="100%" alt="clearml-architecture">
```
Comment (helrewaidy) [inline README.md:215]: remove this image and include your diagrams the you used in the report instead

File: README.md (hunk 2 above)
Comment (helrewaidy) [reply README.md:127]: please remove the Black package.

**Review by helrewaidy — COMMENTED** [review]
few more comments

File: README.md [hunk 6]
```diff
@@ -1,2 +1,353 @@
...
+````
+
+#### [Start using ClearML](https://clear.ml/docs/latest/docs/getting_started/ds/ds_first_steps) 
+
+The ClearML run-time components:
+
+* The ClearML Python Package - for integrating ClearML into scripts by adding just two lines of code, and optionally extending experiments and other workflows with ClearML's powerful and versatile set of classes and methods.
+* The ClearML Server - for storing experiment, model, and workflow data; supporting the Web UI experiment manager and MLOps automation for reproducibility and tuning. It is available as a hosted service and open source to deploy ClearML Server.
+
+<img src="https://i.imgur.com/purP0Rs.jpeg" width="100%" alt="clearml-architecture">
```
Comment (helrewaidy) [inline README.md:214]: The image resolution is very low; Please increase the resolution and you can upload it to this repo, maybe under resources dir

**Review by helrewaidy — APPROVED** [review]

**Comment by MariamTurky** [comment]
Upload README file

**Pull request merged**